pages/common/jest.md

```diff
@@ -7,11 +7,11 @@
 
 `jest`
 
-- Run the test suites from files whose paths match the given regex patterns:
+- Run the test suites from files whose paths match the given regular expression:
 
 `jest {{test_file1}} {{path/to/test_file2.js}}`
```

Comment:
What?  Where's the "regular expression" (or "regex patterns")?

Reply:
I checked. `test_file1` is a placeholder for a filename or regex I think we should use:
```suggestion
`jest "{{regex1}}" "{{regex2}}"`
```
Also I think is the history `test_file1` and `test_file2` was misreading.

Reply:
It should be `regular_expression`, not `regex` @navarroaxel, see the PR's title ;)

Reply:
> `test_file1` is a placeholder for a filename or regex

A filename or a file path? If it's a path, does jest search the whole filesystem for a file that matches the regex?

Reply:
@marchersimon Maybe we can split this example in 2, because you can specify a `path/to/test_file` or you can use a `regex` for the filename (or path).
Jest looks up for the filename inside the current directory and sub-directories.

 I'm sure Jest skips the `node_modules` folder, otherwise the search will take too long. :fearful:

Reply:
Done: 09538ee639d3135a22d730dadfe4d49652a5647f

`jest` also has a french version, which would need this change too. @nicokosi would be amazing if you could help out with this one.

Reply:
@marchersimon OK, I'll try do do that.

Reply:
Thanks @nicokosi

Reply:
Done in comments https://github.com/tldr-pages/tldr/pull/5830/files#r620883630 and https://github.com/tldr-pages/tldr/pull/5830/files#r620883337.

Reply:
Thanks a lot, I've commited you suggestions in df3e47dc96a564be956186bc87e94ca5628b57c1, I hope it looks alright.

Reply:
It's perfect! :heavy_check_mark: